docs/src/main/markdown/json.md

```diff
@@ -15,8 +15,17 @@ Make sure that you are using ScalaPB 0.5.x or later.
 In `build.sbt` add a dependency on `scalapb-json4s`:
 
 ```scala
+// For ScalaPB 0.10.x:
+libraryDependencies += "com.thesamet.scalapb" %% "scalapb-json4s" % "0.10.0-M1"
+
+// For ScalaPB 0.9.x:
+libraryDependencies += "com.thesamet.scalapb" %% "scalapb-json4s" % "0.9.3"
```

Comment:
technically `0.10.0` should work from what I understand, but it's confusing if I put it here

Reply:
Agreed. Let's leave at 0.9.3.

Reply:
can't see 0.9.4 on https://github.com/scalapb/scalapb-json4s/releases nor https://mvnrepository.com/artifact/com.thesamet.scalapb/scalapb-json4s